File: Swift/Brute Force/기능개발.swift
func solution(_ progresses:[Int], _ speeds:[Int]) -> [Int] {
    var day = (100 - progresses[0])/speeds[0]
    if (100 - progresses[0])%speeds[0] > 0 { day += 1 }
    var tmp = 0
    var ans = [Int]()
    for i in 0..<progresses.count {
        let progress = progresses[i] + speeds[i]*day
        if progress >= 100 {
            tmp += 1
        }
        else {
            day += (100 - progress)/speeds[i]
            if (100 - progress)%speeds[i] > 0 {
                day += 1
            }
            ans.append(tmp)
            tmp = 1
        }
    }
    ans.append(tmp)
    return ans
}

File: Python/Else/기능개발.py
from collections import deque
def solution(progresses, speeds):
    answer = []
    que = deque()
    for i in range(len(progresses)):
        que.append([progresses[i], speeds[i]])
    days = 0
    days += (100-progresses[0]-speeds[0]*days )//speeds[0]
    if (100-progresses[0])%speeds[0] > 0: days += 1
    tmp = 0
    while(que):
        progress, speed = que.popleft()
        if 100-progress-speed*days <= 0:
            tmp += 1
        else:
            answer.append(tmp)
            tmp = 1
            days += (100-progress-speed*days )//speed
            if (100-progress)%speed > 0: days += 1
    answer.append(tmp)
    return answer
